rust/src/common/error.rs

```diff
@@ -168,8 +169,8 @@ error_messages! { ConnectionError
         24: "Address translation map does not match the server's advertised address list. User-provided servers not in the advertised list: {unknown:?}. Advertised servers not mapped by user: {unmapped:?}.",
     ValueTimeZoneNameNotRecognised { time_zone: String } =
         25: "Time zone provided by the server has name '{time_zone}', which is not an officially recognized timezone.",
-    ValueTimeZoneOffsetNotImplemented { offset: i32 } =
-        26: "Time zone provided by the server has numerical offset '{offset}', which is not yet supported by the driver.",
+    ValueTimeZoneOffsetNotRecognised { offset: i32 } =
+        26: "Time zone provided by the server has numerical offset '{offset}', which is recognised as a valid value for offset in seconds.",
```

Comment:
You mean "_not_ recognised"?
```suggestion
    ValueTimeZoneOffsetNotRecognised { offset: i32 } =
        26: "Time zone provided by the server has numerical offset '{offset}', which is not recognised as a valid value for offset in seconds.",
```

Reply:
Exactly! Thanks